hadoop-ozone/dist/src/main/smoketest/compatibility/read.robot

```diff
@@ -46,3 +46,9 @@ FSO Bucket Can Be Read
     Execute    ozone fs -get ofs://om/vol1/fso-bucket-${SUFFIX}/dir/subdir/file ${TEMP_DIR}/
     Execute    diff -q ${TESTFILE} ${TEMP_DIR}/file
     [teardown]    Execute    rm -f ${TEMP_DIR}/file
+
+HSync Lease Recover Can Be Used
+    Pass Execution If    '${DATA_VERSION}' < '${HSYNC_VERSION}'      Skipped the test case
```

Comment:
It looks like write and read test cases for HSync are independent.  This read test case uses the file created by the FSO test case as input.  Therefore here we do not need skip based on `${HSYNC_VERSION}`, but `${FSO_VERSION}`.

```suggestion
    Pass Execution If    '${DATA_VERSION}' < '${FSO_VERSION}'      Skipped the test case
```